structuralcodes/codes/mc2010/_concrete_creep_and_shrinkage.py

```diff
@@ -0,0 +1,818 @@
+
+    args:
+        rh (float): The relative humidity of the environment.
+        notional_size (float): The notional size of the considered
+            element in mm, defined as 2A/u.
+
+    returns:
+        float: Multiplication factor beta_RH.
+    """
+    _check_RH(rh)
+    if rh < 1:
+        return (1 - rh) / ((0.1 * notional_size / 100) ** (1 / 3))
+    return (1 - rh / 100) / ((0.1 * notional_size / 100) ** (1 / 3))
+
+
+def _calc_beta_dc_t0(t0_adj: float) -> float:
+    """Calculate multiplication factor that accounts for the effect of
+        the (temperature corrected) age of the concrete when loading is
+        applied to calculate the drying creep coefficient.
+
+    Defined in fib Model Code 2010 (2013), Eq. 5.1-70.
+
+    args:
+        t0_adj (float): The temperature corrected age of the concrete
+            when the loading is applied in days, as defined in fib
+            Model Code 2010 (2013). Eq. 5.1-85.
+
+    returns:
+        float: Multiplication factor beta_dc_t0.
+    """
+    return 1 / (0.1 + t0_adj**0.2)
+
+
+def _calc_alpha_fcm(fcm: float) -> float:
+    """Calculate multiplication factor that accounts for the effect of
+        the strength of the concrete to calculate the drying creep
+        coefficient.
+
+    Defined in fib Model Code 2010 (2013), Eq. 5.1-71d.
+
+    args:
+        fcm (float): The mean compressive strength of the concrete in
+            MPa.
+
+    returns:
+        float: Multiplication factor alpha_fcm.
+    """
+    return np.sqrt(35 / fcm)
+
+
+def _calc_beta_h(notional_size: float, alpha_fcm: float) -> float:
+    """Calculate multiplication factor that accounts for the effect of
+        the notional size of the to calculate the drying creep
+        coefficient.
+
+    Defined in fib Model Code 2010 (2013), Eq. 5.1-71c
+
+    args:
+        notional_size (float): The notional size of the considered
+            element in mm, defined as 2A/h.
+        alpha_fcm (float): Multiplication factor that accounts for the
+            effect of the strength of the concrete on the drying creep
+            coefficient.
+
+    returns:
+        float: Multiplication factor beta_h.
+    """
+    return min(1.5 * notional_size + 250 * alpha_fcm, 1500 * alpha_fcm)
+
+
+def _calc_gamma_t0(t0_adj: float) -> float:
+    """Calculate exponent that accounts for the effect of the
+        (temperature corrected) age of the concrete when loaded. Used
+        to calculate the drying creep coefficient.
+
+    Defined in fib Model Code 2010 (2013), Eq. 5.1-71b.
+
+    args:
+        t0_adj (float): The temperature corrected age of the concrete
+            when the loading is applied in days, as defined in fib
+            Model Code 2010 (2013). Eq. 5.1-85.
+
+    returns:
+        float: Exponent gamma_t0.
+    """
+    return 1 / (2.3 + 3.5 / np.sqrt(t0_adj))
+
+
+def _calc_beta_dc_t(
+    time: np.ndarray, t0: float, beta_h: float, gamma_t0: float
```

Comment:
same comment as before if time is for sure a numpy array or can be a float. The output can change accordingly based on user input?